src/server/internal/recalcrequests/index.js

```diff
@@ -0,0 +1,19 @@
+
+function RecalcRequests(cache, requestmonitor) {
+    const DB_KEY = 'totalIncomingRequests';
+
+    async function recalculate() {
+        const requests = await requestmonitor.getRequests();
+        await cache.deleteKey(DB_KEY).then( () => {
+            cache.increment(DB_KEY, requests.length);
+        });
+    }
+
+    return Object.freeze({
+        recalculate
+    });
+};
+
+RecalcRequests.deps = ['cache', 'requestmonitor'];
```

Comment:
If you choose that implement in other service then this is the right solution that this is an IMT and you use other services by deps. 👍